### PR TITLE
Add login form

### PR DESCRIPTION
**Pull request by Kaiserabbas**
- Added login form
- Added the header and paragraph
- Added username and password form control inputs
- Added a submit button

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -275,8 +275,25 @@ <h3 class="im-qaisar">Glad to see you!</h3>
...
+      <form>
+          <label for="name"></label>
+          <input class="input" type="text" id="name" name="name" placeholder="Qaisar Abbas" required><br><br>
+
+          <label for="email"></label>
+          <input class="input" type="email" id="email" name="email" placeholder="kayser.abbas@gmail.com" required><br><br>
 
+          <label for="message"></label>
+          <textarea class="input" id="message" name="message" rows="6" cols="35" placeholder="Write message here" required></textarea><br><br>
+
```
Comment (agneta1991) [inline index.html]: one of the requirements is maximum length of char.
You can add min and max char requirements by minlength="1" maxlength="500"

File: index.html [hunk 2]
```diff
@@ -275,8 +275,25 @@ <h3 class="im-qaisar">Glad to see you!</h3>
             <li> <img src="./public/adobe-illustrator.svg" alt=""> <span>Adobe Illustrator</span> </li>
           </ul>
         </div>
+
     </section>
+    <section class="form">
+      <h1>Contact Me</h1>
+      <p>If you have an application, you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it</p>
+
```
Comment (tsigewossne) [inline index.html]: Please Remove unwanted space from your code base in order to save memory.

File: index.html (hunk 1 above)
Comment (Kaiserabbas) [reply index.html]: Thanks Noted.

File: style.css [hunk 3]
```diff
@@ -303,3 +303,45 @@ body {
...
+  border-top-left-radius: 20%;
+  padding: 10%;
+}
+
+.form h1 {
+  text-align: center;
+  color: #fff;
+  font-size: 2rem;
+  text-decoration: none !important;
+}
```
Comment (yusufyusie) [inline style.css]: Do not overuse !important rule in your CSS style definitions.

**Review by tsigewossne — COMMENTED** [review]
# changes required ♻️ 

-Please take action and make the required changes 

-Thankyou

File: style.css (hunk 3 above)
Comment (Kaiserabbas) [reply style.css]: Noted Thank You

File: index.html (hunk 2 above)
Comment (Kaiserabbas) [reply index.html]: Okay noted

File: index.html [hunk 4]
```diff
@@ -275,8 +275,25 @@ <h3 class="im-qaisar">Glad to see you!</h3>
...
+      <form>
+          <label for="name"></label>
+          <input class="input" type="text" id="name" name="name" placeholder="Qaisar Abbas" required><br><br>
+
+          <label for="email"></label>
+          <input class="input" type="email" id="email" name="email" placeholder="kayser.abbas@gmail.com" required><br><br>
```
Comment (yusufyusie) [inline index.html]: I recommend to use a pattern for email

Comment (Kaiserabbas) [reply index.html]: Thank you for the suggestion.

**Pull request closed without merging**